### PR TITLE
[ENH] add "visual correction" and "head stabilization" metadata

### LINKED ISSUE #1531: Suggestions for new task metadata: HeadStabilization and VisualAids
Talking with @mszinte on BEP20 for eyetracking we identified a couple of metadata that are important (though not required) for eyetracking but could be useful for other modalities.

This issue is just to see how others feel about adding those into events.json:
https://bids-specification.readthedocs.io/en/latest/modality-specific-files/task-events.html

- HeadStabilization: if a head stabilization was used (chin rest, head rest, both, none). This is important in eyetracking as it helps ensuring a constant distance between the screen and the participant and reduces noise during acquisition. It is also important for generic vision psycho physics experiments even if no eyetracking is performed.

- VisualAids: glasses, lenses, none. Once again important to know in eyetracking as it may explain sources of noise. But it is also something that can be added for any data acquisition with a task so it feels like it belongs more to a task generic metadata than something specific to eyetracking.

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #1531 

- adds HeadStabilization to StimulusPresentation in events.json
- adds VisualAids in events.json

### CODE REVIEW

File: src/schema/objects/metadata.yaml [hunk 1]
```diff
@@ -3543,6 +3552,13 @@ VELChannelCount:
     Number of linear velocity channels.
   type: integer
   minimum: 0
+VisualAids:
+  name: VisualAids
+  display_name: Visual aids
+  description: |
+    Type of visual aids worn by the participant during the experiment.
+    Example: "spectacles", "lenses", "none".
```
Comment (effigies) [inline src/schema/objects/metadata.yaml:3560]:
```suggestion
VisualAids:
  name: VisionCorrection
  display_name: Vision correction
  description: |
    Equipment used to correct participant vision during an experiment.
    Example: "spectacles", "lenses", "none".
```

File: src/modality-specific-files/task-events.md [hunk 2]
```diff
@@ -272,7 +272,9 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
         "ScreenDistance": 1.8,
         "ScreenRefreshRate": 60,
         "ScreenResolution": [1920, 1200],
-        "ScreenSize": [0.472, 0.295]
-    }
+        "ScreenSize": [0.472, 0.295],
+        "HeadStabilization": "none"
+    },
+    "VisualsAids": "lenses"
```
Comment (effigies) [inline src/modality-specific-files/task-events.md:278]:
```suggestion
    "VisionCorrection": "lenses"
```

File: src/schema/rules/sidecars/events.yaml [hunk 3]
```diff
@@ -11,3 +11,4 @@ StimulusPresentation:
     - suffix == "events"
   fields:
     StimulusPresentation: recommended
+    VisualAids: optional
```
Comment (effigies) [inline src/schema/rules/sidecars/events.yaml:14]:
```suggestion
    VisionCorrection: optional
```

**Review by effigies — COMMENTED** [review]
"Visual aid" is something that presents information visually. It is rarely-to-never used to refer to corrective eyewear. Would `VisionCorrection` work?

Otherwise, this seems fine.

**Comment by Remi-Gau** [comment]
> "Visual aid" is something that presents information visually. It is rarely-to-never used to refer to corrective eyewear. Would `VisionCorrection` work?

yes that would make sense to me: will ping some of the people on the eyetracking BEP to make sure that it is good with them too.

File: src/schema/objects/metadata.yaml [hunk 4]
```diff
@@ -3543,6 +3552,13 @@ VELChannelCount:
     Number of linear velocity channels.
   type: integer
   minimum: 0
+VisualAids:
```
Comment (sappelhoff) [inline src/schema/objects/metadata.yaml:3555]:
```suggestion
VisionCorrection:
```

**Review by sappelhoff — APPROVED** [review]

**Review by effigies — APPROVED** [review]
Works for me. I assume the eye tracking bep has had a chance to object.

**Pull request merged**